apps/meteor/server/models/raw/LivechatVisitors.ts

```diff
@@ -32,6 +32,7 @@ export class LivechatVisitorsRaw extends BaseRaw<ILivechatVisitor> implements IL
 			{ key: { username: 1 } },
 			{ key: { 'contactMananger.username': 1 }, sparse: true },
 			{ key: { 'livechatData.$**': 1 } },
+			{ key: { activity: 1 } },
```

Comment:
I wonder if it would be better to use a sparse index instead. 🤔

Reply:
Huh, this should had a partialFilterexp 🤔